React/src/Pages/Orders.tsx

```diff
@@ -0,0 +1,5 @@
+const Orders = () => {
+    return <div>Orders</div>
+}
+
+export default Orders;
```

Comment:
On master we have Orders list working with api

Reply:
Oj bedzik bedzik